### ISSUE TITLE
Cannot add/modify display/variable lists within self-managed

### ISSUE THREAD

**Issue by Coryv221** [PardeeCenterDU/IFs-Issues-Tracking#24]
Date found: 1/28/2021
IFs Version: 7.62 dotNET
Found by: Kaylin McNeil
Error Type: missing features

Description: Cannot add/modify display lists within self-managed

Steps to Replicate: W/in self-managed, variable selection options, no 'create variable/display list from selection status box'

Reproducible in VB6 (Y/N): NA

**Comment by PardeeCenterIFs** [comment]
Right, this is one of those Access related features that we're waiting for a stronger comitment to .NET, idealy would translate once we move everything away from Access, so away from vb6.

**Comment by PardeeCenterIFs** [comment]
We could start simply with display variable lists from here, even if we can't create them yet.

**Comment by Coryv221** [comment]
Completed work on displaying the variable lists, now working on making them correctly load into the grid of selected variables
